docs/releases/4.1.txt

```diff
@@ -156,6 +156,17 @@ CSRF
 
 * ...
 
+Database backends
+~~~~~~~~~~~~~~~~~
+
+* Third-party database backends can now specify the minimum required version
+  of the database using ``DatabaseFeatures.minimum_database_version``
+  attribute.
```

Comment:
add: which is a tuple (e.g. `(10,)` means "10.0").

and add (cominee next bullet):

"If a minimum version is specified, backends must also implement `DatabaseWrapper.get_database_version()`, which returns a tuple of the current database version.

Reply:
I've mixed two bullets and added your note